src/Routes/RosPage/RosPage.js

```diff
@@ -198,14 +194,32 @@ class RosPage extends React.Component {
             : [];
     }
 
+    setColumnModalOpen = (modalState) => {
+        this.setState({
+            isColumnModalOpen: modalState
+        });
+    }
+
+    getActiveColumns = () => {
+        const { columns } = this.props;
+        return columns.filter(column => column.isChecked);
+    }
+
     renderConfigStepsOrTable() {
         const { state: SFObject } = CUSTOM_FILTERS;
+        const activeColumns = this.getActiveColumns();
 
         return (
             this.props.showConfigSteps
                 ?   <ServiceNotConfigured/>
                 :   <Card className='pf-t-light  pf-m-opaque-100'>
                     <CardBody>
+                        <ManageColumnsModal
+                            isModalOpen={this.state.isColumnModalOpen}
+                            setModalOpen={this.setColumnModalOpen}
+                            activeColumns={this.props.columns}
```

Comment:
@PreetiW Could you rename this prop to something else (as discussed) so as to avoid confusion.

Reply:
Thanks @r14chandra for the review, I have refactored the prop name and have pushed the changes.